CMakeLists.txt

```diff
@@ -148,6 +148,37 @@ function(append VAR)
   set(${VAR} ${${VAR}} ${ARGN} PARENT_SCOPE)
 endfunction()
 
+function(link_opengl)
+    if (PLATFORM_MAC)
+      omim_link_libraries(
+        ${PROJECT_NAME}
+        "-framework OpenGL"
+      )
+    endif()
+
+    if (PLATFORM_LINUX)
+      omim_link_libraries(
+        ${PROJECT_NAME}
+        ${OPENGL_gl_LIBRARY}
+      )
+    endif()
+endfunction()
+
+function(link_qt5_core)
+  omim_link_libraries(
+    ${PROJECT_NAME}
```

Comment:
It's not a good idea to implicitly capture PROJECT_NAME here. Much better to pass project name as an argument.